ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -121,12 +174,48 @@ export const CoinOverview = ({
     if (isEvm) {
       if (showNativeTokenAsMainBalance) {
         return (
-          <PercentageAndAmountChange
-            value={tokensMarketData?.[zeroAddress()]?.pricePercentChange1d}
-          />
+          <Box className="wallet-overview__currency-wrapper">
+            <PercentageAndAmountChange
+              value={tokensMarketData?.[zeroAddress()]?.pricePercentChange1d}
+            />
+            <ButtonLink
+              endIconName={IconName.Export}
+              endIconProps={{
+                size: IconSize.Sm,
+                color: IconColor.primaryDefault,
+                className: 'wallet-overview__portfolio_button',
+              }}
+              size={ButtonLinkSize.Inherit}
+              onClick={handlePortfolioOnClick}
+              as="a"
+              data-testid="portfolio-link"
+              className="wallet-overview__portfolio_text"
+            >
+              {t('portfolio')}
+            </ButtonLink>
+          </Box>
         );
       }
-      return <AggregatedPercentageOverview />;
+      return (
+        <Box className="wallet-overview__currency-wrapper">
+          <AggregatedPercentageOverview />
+          <ButtonLink
+            endIconName={IconName.Export}
+            endIconProps={{
+              size: IconSize.Sm,
+              color: IconColor.primaryDefault,
+              className: 'wallet-overview__portfolio_button',
```

Comment:
Can remove this styling as well since the designs should be using design system ButtonLink component and not needing the additional spacing

Reply:
removed wallet-overview__portfolio_button and used `margin: 2` in `endIconProps` instead